### ISSUE TITLE
2.7.7 Patch Release

### ISSUE THREAD

**Issue by danahertzberg** [az-digital/az_quickstart#2965]
- #2880
- #2908
- #2900 
- #2945
- ~#2956~
- #2934
- #2977
- #2969
- #3009

## Pick list
```
git cherry-pick a1c2231c209951cde631cfedf4946e47cd0295c4
git cherry-pick 6de0887adb399b9f9d11f6b70b070d2d59229c83
git cherry-pick 88202ebf3e4aa6d8e7b3989f91179679d660a95b
git cherry-pick 6f5d7b011453b46ef3557ff52cab7da980704558
git cherry-pick e20b456a98cd5061b2f600bed0fc37b5a1d35d09
git cherry-pick 51cb17d95052d1a9d2cb21cda87508ac46c685bc
git cherry-pick 7bda10491b322f808803a4f94325f59fd22f95a1
git cherry-pick f29714ab87dabca0db1fb7ad69946bee0343b500
```

**Comment by joeparsons** [comment]
Removed #2956 because that fix was only needed in 2.8.x.

**Comment by joeparsons** [comment]
## Release notes snippet

`git log 2.7.6..HEAD --pretty='format:- %s'`
```
- Closes #3008: Update seboettg/citeproc-php requirement from 2.6.0 to 2.6.2 (#3009)
- Closes #2852 Deprecation warnings while news feed is importing (#2969)
- Closes #1675 Minor update to help text for "Where" section of Event Page (#2977)
- Close #2933 Update external link field help text to mention University calendar, and to be more concise. (#2934)
- Closes #2944 Update style.css text field widths (#2945)
- Closes #2102 Error upon save when linking to a document in a Card Link URL field (#2900)
- Closes #2907 Make Manual Migration Lookup DB configurable. (#2908)
- Closes #2671: Enable copy options based on content type (#2880)
- Back to dev.
```

**Comment by joeparsons** [comment]
Release created: https://github.com/az-digital/az_quickstart/releases/tag/2.7.7
